lib/ui/discover/discover_child_item.dart

```diff
@@ -7,81 +7,58 @@ import 'package:farmsmart_flutter/utils/styles.dart';
 import 'package:flutter/material.dart';
 import 'package:farmsmart_flutter/utils/assets.dart';
 
+import 'discover_page.dart';
+
 class DiscoveryListItem {
```

Comment:
naming here: I'm asking myself; what are we rendering? - I think a list of articles... So I would name this ArticleListItem.   The fact its in a discovery section is a design detail we may change (maybe we want the articles on the home screen after user testing).

Reply:
note: remember to rename all the other things that have discover..

Reply:
Cool 👍🏻renaming